docs/ref/contrib/humanize.txt

```diff
@@ -34,11 +34,13 @@ You can pass in either an integer or a string representation of an integer.
 ``intcomma``
 ============
 
-Converts an integer to a string containing commas every three digits.
+Converts an integer or a float to a string containing commas every three
```

Comment:
I'd suggest "Converts an integer or float (or a string representation of either) ..." and remove the sentence later on " You can pass in either an integer or a string representation of an integer.".